backend/src/controllers/v2/secretsController.ts

```diff
@@ -690,7 +691,7 @@ export const getSecrets = async (req: Request, res: Response) => {
     }   
     */
 
-  const { tagSlugs, secretPath } = req.query;
+  const { tagSlugs, secretPath, include_imports } = req.query;
```

Comment:
I don't think any of our clients are using v2 any more. I'll be sending out a deprecation notice for this soon